src/lib/constants.js

```diff
@@ -102,7 +102,7 @@ export const PDFJS_WIDTH_PADDING_PX = 40; // Should match SCROLLBAR_PADDING in p
 export const PDFJS_HEIGHT_PADDING_PX = 5; // Should match VERTICAL_PADDING in pdf_viewer.js
 
 export const DOC_LEGACY_STATIC_ASSETS_VERSION = '2.76.0'; // IE11 only
-export const DOC_STATIC_ASSETS_VERSION = '2.100.0'; // Modern browsers
+export const DOC_STATIC_ASSETS_VERSION = '4.2.67'; // Modern browsers
```

Comment:
Historically, we've based these versions on the version of Preview SDK for which the new assets are being introduced.

Reply:
Yes sorry about that. What I sent was not for production. Yes let's stick with the way we currently do it.